packages/valory/connections/openai/connection.yaml

```diff
@@ -19,6 +19,7 @@ config:
   max_tokens: 500
   openai_api_key: null
   temperature: 0.7
+  request_timeout: 300
```

Comment:
Please override `request_timeout` at the agent and the service as well.

Reply:
Fixed